src/content/community/get-involved/open-research/index.md

```diff
@@ -0,0 +1,413 @@
+
+
+#### Recent research
+
+[Handling withdrawals from Lido](https://ethresear.ch/t/handling-withdrawals-in-lidos-eth-liquid-staking-protocol/8873)
+
+[Withdrawal credentials](https://ethresear.ch/t/withdrawal-credential-rotation-from-bls-to-eth1/8722)
+
+### DeFi
+
+Decentralized finance (DeFi) is one of the primary classes of application built on top of Ethereum. DeFi aims to create composable "money legos" that allow users to store, transfer, lend, borrow and and invest crypto-assets using smart contracts. This is a fast-moving space that is constantly updating. Research into secure, efficient and accessible protocols is constantly needed.
+
+#### Background reading
+
+[DeFi](https://ethereum.org/en/defi/)
+
+[Coinbase: What is DeFi?](https://www.coinbase.com/learn/crypto-basics/what-is-defi)
+
+
+#### Recent research
+
+[Decentralized finance, centralized ownership?](https://arxiv.org/pdf/2012.09306.pdf)
+
+[Optimism: The road to sub-dollar transactions](https://medium.com/ethereum-optimism/the-road-to-sub-dollar-transactions-part-2-compression-edition-6bb2890e3e92)
+
+### Fee markets
+
+Ethereum transaction fees protect the network from spam, denial-of-service attacks and bad smart-contract code. However, it also affects the end-user by influencing the cost of interacting with Ethereum. Balancing network security with end-user user-experience is an ongoing challenge that requires further research and development. 
+
+#### Background reading
+
+[EIP-1559 (Robust Incentives Group)](https://ethereum.github.io/abm1559/notebooks/eip1559.html)
+
+#### Recent research
+
+[EIP-4396](https://eips.ethereum.org/EIPS/eip-4396)
+
+[Multidimensional EIP-1559](https://ethresear.ch/t/multidimensional-eip-1559/11651)
+
+[Making EIP-1559 more like an AMM curve](https://ethresear.ch/t/make-eip-1559-more-like-an-amm-curve/9082)
+
+[Cross domain MEV](http://arxiv.org/abs/2112.01472)
+
+
+## Clients and protocol research
+
+### Consensus clients
+
+The consensus layer is concerned with the proof-of-stake mechanism securing Ethereum from the merge onwards. There is abundant ongoing research and development relating to PoS concepts and implementation, for example identifying and patching vulnerabilities, quantifying cryptoeconomic security, increasing the security or performance of client implementations, building light clients and developing staking apps. Some fundamental redesigns of the protocol to confer large improvements to Ethereum are also being researched, for example single slot finality. Furthermore, the efficiency and safety and monitoring of the peer-to-peer networking between consensus clients are also important research topics.
+
+#### Background reading
+
+[Casper-FFG paper](https://arxiv.org/abs/1710.09437)
+
+[Casper-FFG Explainer](https://arxiv.org/abs/1710.09437)
+
+[Gasper paper](https://arxiv.org/abs/2003.03052)
+
+
+##### Client implementations
+
+[Prysm client](https://docs.prylabs.network/docs/how-prysm-works/beacon-node/)
```

Comment:
I think we should just link to here for list of execution client implementations: https://ethereum.org/en/developers/docs/nodes-and-clients/#consensus-clients

This allows us to focus on only one place we ever need to keep things up to date.

Reply:
Do you think it works best as a link out or an embed? Thinking about the ux of leaving the page, opening new tab or reading in place, but need to weigh consistency of presentation.